cpp/src/parquet/file_deserialize_test.cc

```diff
@@ -814,6 +834,30 @@ TEST_F(TestPageSerde, DictCrcCheckNonExistent) {
                          /* verification_checksum */ true, /* has_dictionary */ true);
 }
 
+TEST_F(TestPageSerde, DataPageV2CrcCheckSuccessful) {
```

Comment:
Should we also add test with `has_dictionary=true` for completeness?

Reply:
nit: switching to parameterized test would be more favorable here.

Reply:
> Should we also add test with `has_dictionary=true` for completeness?

Sure, let me try it